afen/src/main.c

```diff
@@ -31,29 +30,37 @@ static int r_afen_parse(RParse *p, const char *data, char *str) {
 	}
 
 	strcpy (str, input);
-	return res;
+	return true;
 }
 
+// RParse plugin Definition Info
+RParsePlugin r_parse_plugin_afen = {
+	.name = "rparse-afen",
```

Comment:
What about naming it “afen”
```suggestion
	.name = "afen",
```